nomination/templates/nomination/project_about.html

```diff
@@ -1,59 +1,107 @@
+                <h4 class="panel-title">Project Details</h4>
+            </div>
+            <div class="panel-body">
+                <ul class="list-group">
+                    <li class="list-group-item col-sm-6">
+                        <b>Project Name:</b> {{ project.project_name}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Project Abbreviation:</b> {{ project.project_slug}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Project Starting Date:</b> {{ project.project_start|date:"M d, Y"}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Project Ending Date:</b> {{ project.project_end|date:"M d, Y"}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Nomination Starting Date:</b> {{ project.nomination_start|date:"M d, Y"}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Nomination Ending Date:</b> {{ project.nomination_end|date:"M d, Y"}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Number of URLs Nominated:</b> {{ url_count }}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Number of Nominators:</b> {{ nominator_count }}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Admin Name:</b> {{ project.admin_name}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Admin E-mail:</b> {{ project.admin_email}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Project URL:</b> {{ project.project_url}}
+                    </li>
+                    <li class="list-group-item col-sm-6">
+                        <b>Registration Required for Project:</b> {{ project.registration_required|yesno:"Yes,No" }}
+                    </li>
+                </ul>
+            </div>
+        </div>
+        {% if show_bookmarklets %}
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <h4 class="panel-title">Project Bookmarklet</h4>
+                </div>
+                <div class="panel-body">
+                    <p>
+                        A bookmarklet is a brief javascript program that can be stored as a bookmark
+                        or favorite in your browser. By following the instructions below you may add
+                        a bookmarklet to your toolbar that allows you to easily nominate URLs to
+                        this project while you browse the Web.
+                    </p>
+                    <b>Firefox</b>
```

Comment:
A description list might fit the bill here: http://getbootstrap.com/css/#description